src/views/document/utils/boxes/ElevationProfile.vue

```diff
@@ -24,10 +24,17 @@
     </form>
 
     <div ref="graph" class="elevation-profile-chart" />
+
+    <div class="elevation-profile-fullscreen-toggle" @click="toggleFullScreenProfile">
+      <fa-icon :icon="toggleIcon" />
+    </div>
   </div>
 </template>
 
 <script>
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons/faChevronDown';
+import { faChevronUp } from '@fortawesome/free-solid-svg-icons/faChevronUp';
```

Comment:
This is @cbeauchesne's work, but the imports are grouped into `font-awesome-config.js`. Don't know the real impact on bundle size & all.

Reply:
All icons are globally imported and available everywhere. I'm actually surprised it works, because icon prop of `<icon>` should be a string or an array:

* moves imports lines to `src/js/vue-plugins/font-awsome-config.js`
* add the relevant line in this files (see after the import, the how-to is here: https://github.com/c2corg/c2c_ui/wiki/How-to-add-a-new-font-awesome-icon) 
*  this for the function `toggleIcon` : 


```
    toggleIcon() {
        return this.fullScreenHidden ? "chevron-up" : "chevron-down";
```

And IMO, you should not use a function, simply use `<fa-icon :icon="fullScreenHidden ? 'chevron-up' : 'chevron-down'" />` in the template :) 

@brunobesson  even it's never free, FA icons are around few hundred of bytes before compression, it's affordable.

Reply:
It works because it's in the docs :)